assets/blocks/course-outline/lesson-block/edit.js

```diff
@@ -31,7 +31,7 @@ const EditLessonBlock = ( props ) => {
 		clientId,
 		name,
 		className,
-		attributes: { title, id, fontSize },
+		attributes: { title, id, fontSize, draft },
```

Comment:
We just missed that in the JSDoc.

Reply:
Fixed in 435a6fadf61cbe139207813581c27681302c00af